projects/login_with_token_csr_only/server/Cargo.toml

```diff
@@ -5,18 +5,18 @@ edition = "2021"
 publish = false
 
 [dependencies]
-api-boundary = "=0.0.0"
```

Comment:
There was no crate/project called `api-boundary`, so I found out it should be `boundary-api` that is available in `crates.io`. Please correct me if I'm wrong.

Reply:
Rather, it's the `api-boundary` crate in this project's workspace.

Reply:
I got confused because of the versioning. Now it's correctly referring to the workspace `api-boundary` crate.